pontoon/settings/base.py

```diff
@@ -277,8 +280,27 @@ def _default_from_email():
     "django_ace",
 )
 
-# A list of IP addresses to be blocked from accessing the app, because they are DDoS'ing the server
-BLOCKED_IPS = os.environ.get("BLOCKED_IPS", "").split(",")
+# A list of IP addresses or IP ranges to be blocked from accessing the app,
+# as a DDoS countermeasure.
+blocked_ip_settings = os.environ.get("BLOCKED_IPS", "").split(",")
+BLOCKED_IPS = []
+BLOCKED_IP_RANGES = []
+for ip in blocked_ip_settings:
+    ip = ip.strip()
+    if ip == "":
```

Comment:
This wasn't obvious to me, but noticed the log always showed the warning with an empty IP. 

Turns out splitting an empty string results in a list with an empty string 🤷🏼 

```
>>> "".split(",")
['']
```

I could check before running the for cycle, but maybe it's useful to keep it here, for example if the list ends with a comma.

```
"a,b,".split(",")
['a', 'b', '']
```

Also noticed that we weren't stripping the list item, so added that as well.